messages/user.json

```diff
@@ -1,9 +1,10 @@
 {
-    "orgRequired": "An org instance is required.",
-    "userQueryFailed": "Failed to query for the user %s.",
-    "invalidHttpResponseCreatingUser": "An invalid http response code (%s) was received while trying to create a user.",
-    "missingFields": "The fields parameters is undefined.",
-    "missingId": "The Salesforce id for the user is required.",
-    "problemsDescribingTheUserObject": "Problems occurred while attempting to describe the user object",
-    "permsetNamesAreRequired": "The permission set names are missing but required."
+  "orgRequired": "An org instance is required.",
+  "userQueryFailed": "Failed to query for the user %s.",
+  "invalidHttpResponseCreatingUser": "An invalid http response code (%s) was received while trying to create a user.",
+  "missingFields": "The fields parameters is undefined.",
+  "missingId": "The Salesforce id for the user is required.",
+  "problemsDescribingTheUserObject": "Problems occurred while attempting to describe the user object",
+  "permsetNamesAreRequired": "The permission set names are missing but required.",
+  "complexityOutOffBound": "INVALID_COMPLEXITY: valid values are between 0 and 5"
```

Comment:
```suggestion
  "complexityOutOffBound": "Invalid complexity value. Valid values are between 0 and 5. "
```

Reply:
Also is there a reason this error message is in sfdx-core and not plugin-user, together with lengthOutOfBound?  If possible I think it's better to keep them all together, unless there's a good reason not to.

Finally, maybe name the variable complexityOutOfBound? (just one "f").